Core/GDCore/Project/ExtensionPropertiesManager.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * GDevelop Core
+ * Copyright 2008-2016 Florian Rival (Florian.Rival@gmail.com). All rights
+ * reserved. This project is released under the MIT License.
+ */
+#ifndef GDCORE_EXTENSIONPROPERTIESMANAGERS_H
+#define GDCORE_EXTENSIONPROPERTIESMANAGERS_H
+#include <map>
+#include "GDCore/Serialization/SerializerElement.h"
+#include "GDCore/String.h"
+
+namespace gd{
+  class Project;
+  class PropertyDescriptor;
+}
+
+namespace gd{
+  class GD_CORE_API ExtensionPropertiesManager {
```

Comment:
Nitpicking: This class is indeed giving some useful functions to manipulate properties, but it's first and mostly the holder of some data which is `std::map<gd::String, std::map<gd::String, gd::String>> properties`. In other words, this thing IS representing/encapsulating the properties. In other words, this thing IS the properties.

So I would simply name this "ExtensionProperties" :)

Reply:
Granted that you have in `gd::PlatformExtension` something that is already called `extensionProperties`, but I think that it is this member in `gd::PlatformExtension` that could have another name.